pkg/symbol/elfutils/debuginfofile.go

```diff
@@ -120,7 +145,8 @@ func (f *debugInfoFile) SourceLines(addr uint64) ([]profile.LocationLine, error)
 			name = getFunctionName(abstractOrigin)
 		}
 
-		file, line := findLineInfo(f.lineEntries[cu.Offset], ch.Ranges)
+		file := f.lineFiles[cu.Offset][ch.Entry.Val(dwarf.AttrCallFile).(int64)].Name
```

Comment:
I'd appreciate some more comments to explain the logic.

Reply:
// An inlined subroutine entry may have these field:
//  - DW_AT_call_line：line number of inlined subroutine call
//  - DW_AT_call_file: File contain inlined subroutine call

Reply:
It would be nice to add this as a comment.